ironic/tests/unit/drivers/modules/drac/test_raid.py

```diff
@@ -1385,3 +1385,68 @@ def test_get_logical_disks(self, mock_list_virtual_disks):
 
         self.assertEqual({'logical_disks': [expected_logical_disk]},
                          props)
+
+    @mock.patch.object(drac_common, 'get_drac_client', spec_set=True,
+                       autospec=True)
+    def test_reset_idrac_configuration(self, mock_get_drac_client):
+        mock_client = mock.Mock()
+        mock_get_drac_client.return_value = mock_client
+
+        with task_manager.acquire(self.context, self.node.uuid,
+                                  shared=False) as task:
+            return_value = task.driver.raid.reset_idrac(task)
+
+            mock_client.reset_idrac.assert_called_once_with(
+                force=True, wait=True)
+
+        self.assertIsNone(return_value)
+        self.node.refresh()
+
+    @mock.patch.object(drac_common, 'get_drac_client', spec_set=True,
+                       autospec=True)
+    def test_reset_idrac_no_change(self, mock_get_drac_client):
+        mock_client = mock.Mock()
+        mock_get_drac_client.return_value = mock_client
+
+        with task_manager.acquire(self.context, self.node.uuid,
+                                  shared=False) as task:
+            return_value = task.driver.raid.reset_idrac(task)
+
+            self.assertEqual(1, mock_client.reset_idrac.call_count)
+
+        self.assertIsNone(return_value)
+        self.node.refresh()
+
+    @mock.patch.object(drac_common, 'get_drac_client', spec_set=True,
+                       autospec=True)
+    def test_known_good_state_configuration(self, mock_get_drac_client):
+        mock_client = mock.Mock()
+        mock_get_drac_client.return_value = mock_client
+
+        with task_manager.acquire(self.context, self.node.uuid,
+                                  shared=False) as task:
+            return_value = task.driver.raid.known_good_state(task)
+
+            mock_client.reset_idrac.assert_called_once_with(
+                force=True, wait=True)
+            mock_client.delete_jobs.assert_called_once_with(
+                job_ids=['JID_CLEARALL_FORCE'])
+
+        self.assertIsNone(return_value)
+        self.node.refresh()
+
+    @mock.patch.object(drac_common, 'get_drac_client', spec_set=True,
```

Comment:
It looks to me like this test is the same as the one above, but the one above also checks the passed parameters.  If this is the case, can you remove this one?

Reply:
I have removed this test case, as we don't makes any changes in raid.py.